src/mbgl/style/style_layer.hpp

```diff
@@ -56,6 +56,7 @@ class StyleLayer : public util::noncopyable {
     float minZoom = -std::numeric_limits<float>::infinity();
     float maxZoom = std::numeric_limits<float>::infinity();
     VisibilityType visibility = VisibilityType::Visible;
+    bool interactive;
```

Comment:
I would’ve called this `isInteractive`, but I don’t know what the prevailing style is in mbgl.